packages/core/src/getStateFromPath.tsx

```diff
@@ -441,6 +454,7 @@ const createConfigItem = (
     regex,
     pattern,
     path,
+    normalizedPath,
```

Comment:
The `normalizedPath` can be calculated from `path` where needed, so there's no need to include it additionally.

Also is it necessary to have both `path` and `normalizedPath`?

Reply:
Thanks for the question! The solution this PR proposes relies on the path being a suffix of the pattern.

Normalizing it in a similar way to pattern helps to make sure that path is always a suffix of the pattern (even if the user entered double or leading/trailing slashes in their configuration). 

What do you think? Is there a better approach? Should we mutate path itself here instead, or clean it up at point of usage?

Reply:
I'm just saying that there's no need to include it in the object when it can be calculated when needed. I have updated the PR with the change.